src/test/resources/data/typicalDataBundle.json

```diff
@@ -198,14 +198,14 @@
       "displayedName": "Instructor",
       "privileges": {
         "courseLevel": {
-          "canviewstudentinsection": true,
-          "cansubmitsessioninsection": true,
-          "canmodifysessioncommentinsection": true,
           "canmodifycourse": true,
-          "canviewsessioninsection": true,
+          "canmodifyinstructor": true,
           "canmodifysession": true,
           "canmodifystudent": true,
-          "canmodifyinstructor": true
+          "canviewstudentinsection": true,
+          "canviewsessioninsection": true,
+          "cansubmitsessioninsection": true,
+          "canmodifysessioncommentinsection": true
```

Comment:
Is it necessary to modify this file, as the instructors below are not modified

Reply:
Just shift the order of the key so that `JSONUtil.toString()` can be used to compare the privileges of instructors in testing.

Reply:
Then how about the privileges of the rest instructors?

Reply:
They are not involved so there is no need to shift them.